### PR TITLE
Update authentication.md

### PR DESCRIPTION
**Pull request by stewart-yu**
Add  some url and signals.

Allow edits from maintainers checkbox

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/5215)

### CODE REVIEW

File: docs/admin/authentication.md [hunk 1]
```diff
@@ -286,7 +286,7 @@ To enable the plugin, configure the following flags on the API server:
...
+| `--oidc-issuer-url` | URL of the provider which allows the API server to discover public signing keys. Only URLs which use the `https://` scheme are accepted.  This is typically the provider's discovery URL without a path, for example "[https://accounts.google.com](https://accounts.google.com)" or "[https://login.salesforce.com](https://login.salesforce.com)".  This URL should point to the level below .well-known/openid-configuration. | If the discovery URL is [https://accounts.google.com/.well-known/openid-configuration](https://accounts.google.com/.well-known/openid-configuration), the value should be [https://accounts.google.com](https://accounts.google.com). | Yes |
```
Comment (Bradamant3) [inline docs/admin/authentication.md]: The original formatting without links is correct -- the URLs should be rendered on the page as strings (values to be provided), not as links. Please revert this change.

File: docs/admin/authentication.md [hunk 2]
```diff
@@ -319,7 +319,7 @@ For an identity provider to work with Kubernetes it must:
...
+A note about requirement #3 above, requiring a CA signed certificate.  If you deploy your own identity provider (as opposed to one of the cloud providers like Google or Microsoft) you MUST have your identity provider's web server certificate signed by a certificate with the `CA` flag set to `TRUE`, even if it is self signed.  This is due to GoLang's TLS client implementation being very strict to the standards around certificate validation.  If you don't have a CA handy, you can use this script from the CoreOS team to create a simple CA and a signed certificate and key pair - [https://github.com/coreos/dex/blob/1ee5920c54f5926d6468d2607c728b71cfe98092/examples/k8s/gencert.sh](https://github.com/coreos/dex/blob/1ee5920c54f5926d6468d2607c728b71cfe98092/examples/k8s/gencert.sh) or this script based on it that will generate SHA256 certs with a longer life and larger key size [https://raw.githubusercontent.com/TremoloSecurity/openunison-qs-kubernetes/master/makecerts.sh](https://raw.githubusercontent.com/TremoloSecurity/openunison-qs-kubernetes/master/makecerts.sh).
```
Comment (Bradamant3) [inline docs/admin/authentication.md]: This change is OK, but would be even more useful if the links were provided more like this:

If you don't have a CA handy, you can use [this script](https://github.com/coreos/dex/blob/1ee5920c54f5926d6468d2607c728b71cfe98092/examples/k8s/gencert.sh) from the CoreOS team to create a simple CA and a signed certificate and key pair. 
Or you can use [this similar script](https://raw.githubusercontent.com/TremoloSecurity/openunison-qs-kubernetes/master/makecerts.sh) generates SHA256 certs with a longer life and larger key size.

Add a paragraph break before the sentences that include the script links, too -- easier to read and find.

File: docs/admin/authentication.md (hunk 1 above)
Comment (stewart-yu) [reply docs/admin/authentication.md]: Thanks. But when you edit origina page, then preview it,  thoes all are links.

File: docs/admin/authentication.md (hunk 2 above)
Comment (stewart-yu) [reply docs/admin/authentication.md]: OK
